lib/resty/core/response.lua

```diff
@@ -226,6 +228,18 @@ local function get_resp_header(tb, key)
 end
 
 
+local function bypass_if_checks()
+    local r = get_request()
+    if not r then
+        error("no request found")
+    end
+    return C.ngx_http_lua_ffi_bypass_if_checks(r)
```

Comment:
I think we do not need any return value here.

Reply:
What about also removing the `return NGX_OK;`
in https://github.com/openresty/lua-nginx-module/pull/2401/files#diff-ea7694db95d4b4f2da2951cf882953a64e545494e6b9302cb9e80d0abf8f8fbbR4562?

Reply:
done